<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>111</width>
      <height>36</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cyan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>yellow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>purple</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>white</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>black</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>106</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>106</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
